packages/@aws-cdk/aws-cloudfront/test/oai.test.ts

```diff
@@ -1,69 +1,69 @@
-import { expect } from '@aws-cdk/assert';
+import '@aws-cdk/assert/jest';
 import * as cdk from '@aws-cdk/core';
-import { nodeunitShim, Test } from 'nodeunit-shim';
 import { OriginAccessIdentity } from '../lib';
 
-/* eslint-disable quote-props */
+test('Origin Access Identity with automatic comment', () => {
```

Comment:
Can you instead put a top-level `describe` here, like `describe('Origin Access Identity', () => ...`? This will make the diff much better 🙂

Reply:
I actually originally did that, but then decided against it. The diff was only marginally better, and there's only 3 existing tests here. I think it's fairly reasonable to review as-is (60 lines, 4 tests total), vs having to keep an extra "describe" around forever for the sake of a simpler diff.

Reply:
What's the problem with having an extra `describe` here...? In fact, when I write new Jest tests, I almost always place them under a top-level `describe` anyway...

Reply:
Is the motivation behind `describes` to produce cleaner diffs though? I'm in favour of adding it for consistency in test suite organization.

I always introduce it when adding new tests to group related tests. Often, this results in only one describe in a test file because that's what best describes the test suite.

Reply:
> I always introduce it when adding new tests to group related tests.

I agree with this, and that's generally how I've used describe -- to group related tests. However, when the group is "the class being tested", I feel that that's already covered by the filename. If literally every test in the file is going to be under the heading of `describe('Class', () => {` then you're not adding any new information/clarity (IMO) and you're losing one level of indentation.

> I'm in favour of adding it for consistency in test suite organization.
> In fact, when I write new Jest tests, I almost always place them under a top-level describe anyway...

😆  I can't imagine any way in which our test suites are consistent, but I'm clearly out-voted here. I'll add a `describe`.

Reply:
you're right, our test suites are not consistent at all.... but we'll get there!

Reply:
I hate top-level describes! They're pointless.